scriptmodules/libretrocores/lr-duckstation.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="lr-duckstation"
+rp_module_desc="PlayStation emulator - Duckstation for libretro"
+rp_module_help="ROM Extensions: .exe .cue .bin .chd .psf .m3u .pbp\n\nCopy your PlayStation roms to $romdir/psx\n\nCopy compatble BIOS files to $biosdir"
+rp_module_licence="PROP"
+rp_module_section="exp"
+rp_module_flags="!all !armv6 aarch64 arm 64bit"
```

Comment:
The arm flag will override the !armv6 for armv6.

Reply:
yup :( how about now? if >armv8 becomes supported I guess we would have to change the script but i can't think of a better way of excluding <armv7

Reply:
Might have been simpler to just put the `!armv6` at the end.

Also could use "neon" as only armv7+ has it to avoid having armv7 / armv8

Reply:
oh I didn't realise the conditions were applied incrementally 👍  i guess `!all arm !armv6 aarch64 64bit` makes sense then

Reply:
Should be fine yeah.

Reply:
done!